PmlUnit/CodeEditor.cs

```diff
@@ -274,6 +277,22 @@ protected override IEnumerable<string> GetExtraArguments(string fileName, int li
         }
     }
 
+    class PMLStudioCodeEditor : BaseCodeEditor
+    {
+        public PMLStudioCodeEditor(string fileName, string arguments)
+            : base(fileName, arguments)
+        {
+        }
+
+        protected override IEnumerable<string> GetExtraArguments(string fileName, int lineNumber)
+        {
+            if (lineNumber > 0)
+                return new string[] { fileName };
+            else
+                return new string[] { fileName };
```

Comment:
remove condition if both branches do exactly the same
```suggestion
            return new string[] { fileName };
```